### ISSUE TITLE
Bug: globby -> fast-glob regression

### ISSUE THREAD

**Issue by therealpecus** [dennisreimann/uiengine#56]
**Describe the bug**
A fresh install fails to build the demo project.

**To Reproduce**
1. follow the quick start:
```
mkdir uiengine-demo && cd uiengine-demo && npm init -y
npm install @uiengine/core @uiengine/adapter-html --save-dev
npx uiengine init --demo
npx uiengine build --serve --watch
```
2. the build fails with:
```
🚨  Build failed!

Child page "elements" does not exist for page "patterns".

Here is a list of available child pages:
```
notably, the list of available child pages is empty.

**Expected behavior**
The build succeeds

**Additional context**
Tried a fresh install with Node v10, v12, v13, same results. This **did not happen** in the past and does not seem to be tied to version 2.6.x. Tried with v2.5.9 and it still breaks. This seems to be a problem with a downward dependency.

**EDIT**: found the culprit in fast-glob. Version 3.1.1 works fine. version 3.2.2 fails.
Quick workaround:
```
rm -rf node_modules/fast-glob
npm i fast-glob@3.1.1
``` 

Any course of action to avoid this?
Thanks in advance,
Matteo

**Comment by dennisreimann** [comment]
Hey Matteo, thanks for reporting and documenting the issue. I know @naltatis also came across this. I'd like to fix it be referencing a fixed fast-glob version, but there is nothing published yet.

> Any course of action to avoid this?

For now I recommend to add fast-glob@3.1.1 as a dependency for you app, so that you do not have to rely on a new UIengine release fixing this. 

Nevertheless I'll see what I can do with the next regular update I'm trying to publish soon.

**Comment by therealpecus** [comment]
Thanks Dennis,

it seems to have broken havoc in quite some places!
Adding a stable version of fast-glob seems the best choice.

Looking forward to the next update! 🎸

**Comment by dennisreimann** [comment]
The freshly released v3.0.0 contains a fix for this problem :)
